src/components/GameCard.vue

```diff
@@ -13,10 +14,12 @@ const imgAlt = computed(() => {
 </script>
 
 <template>
-    <figure class="gamecard">
-        <img class="gamecard_image" :src="imageUrl" :alt="imgAlt" loading="lazy" />
-        <figcaption>{{ name }}</figcaption>
-    </figure>
+    <a :href="'/game/' + id">
```

Comment:
one more suggestion, can you add a style to hide the underline for the anchor tag

```css
a {
    text-decoration: none;
}